app/renderers/barcode_attribute_renderer.rb

```diff
@@ -0,0 +1,5 @@
+class BarcodeAttributeRenderer < Hyrax::Renderers::AttributeRenderer
+  def label
```

Comment:
Let me see if I can figure out how to get i18n to work from the Hyrax's Spanish translation file.